README.md

```diff
@@ -870,6 +871,51 @@ The following props are part of each element in `items`:
 - `onReorderClick: (index, newIndex) => (event) => void`: Returns a function that swaps the items at `index` with `newIndex`.
 - `readonly`: A boolean value stating if the array item is readonly.
 
+### Object Field Template
+
+Similarly to the `FieldTemplate` you can use an `ObjectFieldTemplate` to customize how your
+objects are rendered.
+
+```jsx
+function ObjectFieldTemplate(props) {
+  return (
+    <div>
+      {props.title}
+      {props.description}
+      {props.properties.map(element => <div className="property-wrapper">{element.children}</div>)}
+    </div>
+  );
+}
+
+render((
+  <Form schema={schema}
+        ObjectFieldTemplate={ObjectFieldTemplate} />,
+), document.getElementById("app"));
+```
+
+Please see [customObject.js](https://github.com/mozilla-services/react-jsonschema-form/blob/master/playground/samples/customObject.js) for a better example.
+
+The following props are passed to each `ObjectFieldTemplate`:
+
+- `DescriptionField`: The generated `DescriptionField` (if you wanted to utilize it)
+- `TitleField`: The generated `TitleField` (if you wanted to utilize it).
```

Comment:
I'm kind of ambivalent about passing these field components to the template. Client code already has the opportunity to override the entire template, using any `DescriptionField` and `TitleField` definition they want, and they can re-import DescriptionField and TitleField themselves if they want them. So what's the value?

I don't think `generated` is the right word here -- it's the one taken from the registry.

How about changing `if you wanted to` to `in case you wanted to`?

Reply:
I agree that passing `DescriptionField` and `TitleField` is useless. However, `ArrayField` also passes them to `ArrayFieldTemplate`, so at least we are consistenly useless :)

I copy pasted the descriptions from `ArrayFieldTemplate` manual entry. I changed the descriptions in both manual entries now to "The `DescriptionField/TitleField` from the registry (in case you wanted to utilize it)"